app/views/schools/dashboard/show.html.erb

```diff
@@ -5,66 +5,64 @@
-            <td class="govuk-table__cell"><%= govuk_link_to "View details", schools_cohort_path(cohort_id: school_cohort.cohort.start_year), class: 'app-govuk-link' %></td>
-          </tr>
-          <% if school_cohort.core_induction_programme? %>
-            <tr class="govuk-table__row">
-              <th scope="row" class="govuk-table__header">Materials</th>
-              <td class="govuk-table__cell"><%= school_cohort.core_induction_programme&.name %></th>
-              <% if school_cohort.core_induction_programme_id? %>
-                <td class="govuk-table__cell"><%= govuk_link_to 'Change', schools_core_programme_materials_path(cohort_id: school_cohort.cohort.start_year) %></td>
-              <% else %>
-                <td class="govuk-table__cell"><%= govuk_link_to 'Add', info_schools_core_programme_materials_path(cohort_id: school_cohort.cohort.start_year) %></td>
-              <% end %>
-            </tr>
-          <% elsif school_cohort.full_induction_programme? %>
-            <tr class="govuk-table__row">
-              <th scope="row" class="govuk-table__header">Training provider</th>
-              <td class="govuk-table__cell"><%= school_cohort.lead_provider&.name %></td>
-              <% if school_cohort.lead_provider.present? %>
-                <td class="govuk-table__cell"></td>
-              <% else %>
-                <td class="govuk-table__cell"><%= govuk_link_to 'Sign up', schools_partnerships_path(cohort_id: school_cohort.cohort.start_year) %></td>
-              <% end %>
-            </tr>
+      <dl class="govuk-summary-list">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Cohort year</dt>
+          <dd class="govuk-summary-list__value"><%= school_cohort.cohort.start_year %></dd>
+          <dd class="govuk-summary-list__actions"></dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Programme</dt>
+          <dd class="govuk-summary-list__value"><%= t(school_cohort.induction_programme_choice, scope: %i[manage_your_training induction_programmes]) %></dd>
+          <dd class="govuk-summary-list__actions"><%= govuk_link_to "View details", schools_cohort_path(cohort_id: school_cohort.cohort.start_year), class: 'app-govuk-link' %></dd>
+        </div>
+        <% if school_cohort.core_induction_programme? %>
+          <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key">Materials</dt>
+            <dd class="govuk-summary-list__value"><%= school_cohort.core_induction_programme&.name %></dd>
+            <% if school_cohort.core_induction_programme_id? %>
+              <dd class="govuk-summary-list__actions"><%= govuk_link_to 'Change', schools_core_programme_materials_path(cohort_id: school_cohort.cohort.start_year) %></dd>
```

Comment:
How do these play with screenreader? Do they need some visually hidden helper text?

Reply:
Hi @ethanmills, Screenreader handles the summary-list fine. I've checked elsewhere in the app and can't see we are using any hidden helper text with summary-lists - but if you think this should go in, point me in the direction of an example we should follow